src/webassets/filter/sass.py

```diff
@@ -159,8 +161,7 @@ def _apply_sass(self, _in, out, cd=None):
 
             out.write(stdout.decode('utf-8'))
         finally:
-            if cd:
-                os.chdir(old_dir)
+            pass
```

Comment:
Indeed. I should have taken it out immediately instead of being lazy. Now removed.